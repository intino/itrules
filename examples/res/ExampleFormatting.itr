def type(Person)
    Name: $Name+Uppercase

    Height: $Height ($Height+Words) cm
    Salary: $$$Salary+Separators
end