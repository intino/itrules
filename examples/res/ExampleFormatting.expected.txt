Name: PAU GASOL

Height: 213 (two hundred and thirteen) cm
Salary: $19,285,850